index.bs

```diff
@@ -922,6 +931,12 @@ optionally evidence of [=user consent=] to a specific transaction.
         authenticators are eligible to participate in a {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}}
         operation. See [[#attachment]] for a description of the attachment values and their meanings.
 
+    :   <dfn>requireResidentKey</dfn>
+    ::  This member describes the [=[RPS]=]' requirements regarding availability of the [=credential private keys=]. If the
+        parameter is set to <a>true</a>, the authenticator SHOULD create a [=resident key=] when creating a [=scoped credential=].
```

Comment:
You need to write requirements in an algorithm that actually performs the operations. In this case, it'd be somewhere in authenticatorMakeCredential, not just the non-normative description of the members of MakeCredentialOptions.

I suspect you should s/SHOULD/MUST/. Is there a point in passing `requireResidentKey` if authenticators are still allowed to store the key in the credential ID?